grails-app/controllers/au/org/ala/biocollect/merit/ProjectController.groovy

```diff
@@ -825,7 +872,17 @@ class ProjectController {
             trimmedParams.status = null
         }
 
+        boolean isAlaAdmin = userService.userIsAlaAdmin()
+
+        if (!isAlaAdmin)
+            fq.push('projLifecycleStatus:published')
+
         if (trimmedParams.isUserPage) {
+            if (!fq.contains('projLifecycleStatus:published'))
```

Comment:
Are you trying to get both published and unpublished here? Not adding any `fq` will get both projects.